src/components/ui/Person.vue

```diff
@@ -1,18 +1,15 @@
 <template>
   <div class="person">
     <UserPicture :avatar="{ picture, username }" :size="40" :isStaff="true" />
-    <RouterLink
-      :to="{ name: 'Profile', params: { username } }"
-      :title="`${firstName} ${lastName}`"
-    >
+    <a v-bind:href="personLink" :title="`${firstName} ${lastName}`">
```

Comment:
We should not use `<a>`s.
The page is jumping because the `ShowMore` (my least favorite component) is opened (stays open) when the profile changes. The way this should be handled is setting:
```js
        :closeOnUpdate="true"
        :updateIndicator="JSON.stringify(collection)"
```
in the `ShowMore` component in `ReportingStructureGroup`.
The update indicator is super ugly, maybe we should pass the username down there (or retrieve it from the router, which would be even hackier).
Anyhow, before we clean all of this up I'd be happier with keep the `RouterLink`. Thoughts?